opencensus/metrics/export/gauge.py

```diff
@@ -169,6 +169,34 @@ def to_point_value(self):
         return value_module.ValueDouble(self.value)
 
 
+class CumulativePointLong(GaugePointLong):
+    """A `GaugePointLong` that cannot decrease."""
```

Comment:
Consider putting these under `cumulative.py` to be consistent with other languages.

Reply:
Moved in b898f1a.